plugins/modules/packaging/os/pacman_key.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, George Rawlinson <george@rawlinson.net.nz>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: pacman_key
+author:
+- George Rawlinson (@grawlinson) <george@rawlinson.net.nz>
+version_added: "1.1.0"
+short_description: Manage pacman's list of trusted keys
+description:
+- Add or remove gpg keys from the pacman keyring.
+notes:
+- Use full key ID (16 characters) and fingerprint (40 characters) to avoid key collisions.
+- If you specify both the key id and the URL with C(state=present), the task can verify or add the key as needed.
+- By default, keys will be locally signed after being imported into the keyring.
+- If the specified key id exists in the keyring, the key will not be added.
+- data, file, and url are mutually exclusive.
```

Comment:
```suggestion
- I(data), I(file), and I(url) are mutually exclusive.
```
That should better be mentioned in the description for all of these options. People tend to ignore most of the documentation anyway, and the chance of them noticing this part there is a lot better :)